CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * Add prompters to `orq wf submit` command for CE runtime if workspace and project weren't passed explicitly
 
 🥷 *Internal*
+* Use external URI to cluster when using `auto` config inside studio
```

Comment:
question: what's the outcome of this PR for the user? Does it change what needs to be set in Studio? Does it fix an interaction that wasn't working correctly before?